src/data/dialogmote/dialogmoteSagas.ts

```diff
@@ -0,0 +1,26 @@
+import { all, call, put, takeEvery } from "redux-saga/effects";
+import {
+  innkallingOpprettet,
+  OPPRETT_INNKALLING_FORESPURT,
+  oppretterInnkalling,
+  OpprettInnkallingAction,
+  opprettInnkallingFeilet,
+  opprettInnkallingFullfort,
+} from "./dialogmote_actions";
+import { post } from "../../api";
+
+function* opprettInnkalling(action: OpprettInnkallingAction) {
+  yield put(oppretterInnkalling());
+  try {
+    const path = `${process.env.REACT_APP_ISDIALOGMOTE_ROOT}/v1/dialogmote/personident`;
+    yield call(post, path, action.data, action.fnr);
+    yield put(innkallingOpprettet(action.data));
+    yield put(opprettInnkallingFullfort());
+  } catch (e) {
+    yield put(opprettInnkallingFeilet());
```

Comment:
Her burde vi kanskje logge noe dersom det går galt

Reply:
Vi har vel noe generell logging allerede hvis post-kallet feiler i api/index.ts. Tenker vi kan legge på ekstra logging her senere hvis vi får behov for å behandle spesielle feilsituasjoner e.l.